_source/_posts/2022-03-18-programming-security-and-why-rust.md

```diff
@@ -160,4 +160,4 @@ If you want to learn more about Rust and security in general, check out these ad
 - [A Comparison of Cookies and Tokens for Secure Authentication](/blog/2022/02/08/cookies-vs-tokens)
 - [The Things to Keep in Mind about Auth](/blog/2021/10/29/things-to-keep-in-mind-about-auth)
 
-If you liked this tutorial, chances are you'll enjoy the others we publish. Please follow [@oktadev on Twitter](https://twitter.com/oktadev) and [subscribe to our YouTube channel](https://youtube.com/oktadev) to get notified when we publish new developer tutorials.
+If you liked this article, chances are you'll enjoy the others we publish. Please follow [@oktadev on Twitter](https://twitter.com/oktadev) and [subscribe to our YouTube channel](https://youtube.com/oktadev) to get notified when we publish new developer tutorials.
```

Comment:
I actually like tutorial better, especially if it's a conversion post. If it's awareness, I can see why article might be better.

Reply:
This definitely wad not a tutorial post 😬